website/content/en/status/report-2024-07-2024-09/intro.adoc

```diff
@@ -0,0 +1,9 @@
+Here is the third 2024 status report, with ?? entries.
+
+Unfortunately we are late this quarter, just like last quarter.
+As our readers know, many FreeBSD contributors are volunteers, so it is natural that less important deadlines like the status reports ones are not always met.
+Indeed, if you are not a FreeBSD contributor yet, please consider joining us: you could help lighten someone's burden while learning new skills, working side by side with developers all over the world.
```

Comment:
Perhaps make "consider joining us" a link to https://docs.freebsd.org/en/articles/contributing/?